src/js/memory/renderMemory.ts

```diff
@@ -0,0 +1,58 @@
+import { getById } from '../utilities';
+
+export function renderMemory(): void {	
+	const memoryForm = getById('MemoryForm');
+	const attributes = [
+		{ name: 'display', input: true },
+		{ name: 'label', input: false },
+		{ name: 'input', input: true },
+	];
+
+	for (let i = 1; i <= 5; i++) { //i is the stage
+		const row = document.createElement('div');
+		row.id = `memory_stage_${i}`		
+		attributes.forEach(attribute => { 
+			const group = document.createElement('div');
+			group.className = 'btn-group btn-group-sm align-top';
+			group.setAttribute('role', 'group');
+			group.style.width = `${100 / (attributes.length)}%`;
```

Comment:
Ideally we shouldn't have to set widths specifically here. Why was this necessary?